script.js

```diff
@@ -1,4 +1,147 @@
+var winCount = 0;
+var lossCount = 0;
+var inputCount = 0;
+var username;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (inputCount != 0 && isInvalidInput(input)) {
+    return "There are only 3 input options, please try again";
+  }
+
+  if (inputCount == 0) {
+    inputCount = inputCount + 1;
+    return saveUser(input);
+  }
+
+  var computer = generateComputerSelection();
+
+  var winStatus = checkWinStatus(input, computer);
+  inputCount = inputCount + 1;
```

Comment:
does `inputCount` matter besides when it is 0 or 1? is there a difference when `inputCount` is 2 vs 3 vs 4?